examples/chip-tool/BUILD.gn

```diff
@@ -87,13 +87,11 @@ static_library("chip-tool-utils") {
   deps = []
 
   if (config_use_interactive_mode) {
-    sources += [
-      "commands/interactive/InteractiveCommands.cpp",
-      "commands/interactive/WebSocketServer.cpp",
-      "commands/interactive/WebSocketServer.h",
-      "commands/interactive/WebSocketServerDelegate.h",
+    sources += [ "commands/interactive/InteractiveCommands.cpp" ]
+    deps += [
+      "${chip_root}/src/app/tests/suites/server:test_server",
```

Comment:
What do you think of naming targets at least being `${chip_root}/src/app/tests/suites/server:websocket` to be explicit on what this includes?

Reply:
I moved/renamed the component as : `examples/common/websocket-server`